village/src/App.js

```diff
@@ -19,6 +29,7 @@ class App extends Component {
       <div className="App">
         <SmurfForm />
         <Smurfs smurfs={this.state.smurfs} />
+        <Route path = '/' render = {(props) => <smurfs{...props}smurfs={this.state.smurfsData}/>}/>
```

Comment:
remember to be careful with your component names -this should be capitalized Smurfs